src/commands/transaction/mosaicaddressrestriction.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import { command, metadata, option } from 'clime';
+import {
+    Deadline,
+    MosaicAddressRestrictionTransaction,
+    UInt64,
+} from 'nem2-sdk';
+import { AnnounceTransactionsCommand, AnnounceTransactionsOptions } from '../../announce.transactions.command';
+import { TargetAddressResolver } from '../../resolvers/address.resolver';
+import { MaxFeeResolver } from '../../resolvers/maxFee.resolver';
+import { MosaicIdAliasResolver } from '../../resolvers/mosaic.resolver';
+import { RestrictionKeyResolver } from '../../resolvers/restrictionKey.resolver';
+import { RestrictionValueResolver } from '../../resolvers/restrictionValue.resolver';
+import { AddressAliasValidator } from '../../validators/address.validator';
+import { MosaicIdAliasValidator } from '../../validators/mosaicId.validator';
+import { NumericStringValidator } from '../../validators/numericString.validator';
+import { MosaicRestrictionKeyValidator } from '../../validators/restrictionKey.validator';
+
+export class CommandOptions extends AnnounceTransactionsOptions {
+    @option({
+        flag: 'm',
+        description: 'Mosaic identifier or @alias being restricted.',
+        validator: new MosaicIdAliasValidator(),
```

Comment:
Please, remove validators from options. We have dropped them recently from the other commands.

Reply:
Remove validators of all commands? Or just remove validators of `Mosaic Restriction` command?

Reply:
From ``mosaicaddressrestriction`` and ``mosaicglobalrestriction commands``. We decided to remove validators from options recently because we are already validating the options data format inside resolvers.